<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPatientEdit</class>
 <widget class="QDialog" name="dlgPatientEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="gbIdentification">
       <property name="title">
        <string> Identification </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledName"/>
          </item>
          <item>
           <widget class="QLabel" name="lblGender">
            <property name="text">
             <string>Gender :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbGenderMale">
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbGenderFemale">
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lblDateBirth">
            <property name="text">
             <string>Date of birth :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="deDateBirth">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>1900</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblUniqueId">
            <property name="text">
             <string>Unique identification :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledUniqueId"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbAddress">
       <property name="title">
        <string> Address </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="lblCountry">
              <property name="text">
               <string>Country :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="ledCountry"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblCity">
              <property name="text">
               <string>City: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="ledCity"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="lblRegion">
              <property name="text">
               <string>Region :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="ledRegion"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblZip">
              <property name="text">
               <string>Zip :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="ledZip"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblAddress">
            <property name="text">
             <string>Address :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledAddress"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbVisit">
       <property name="title">
        <string> Statistical data </string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lblPatientOrigin">
          <property name="text">
           <string>Place the patient comes from :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbPatientOrigin"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblReasonToVisit">
          <property name="text">
           <string>Reason of visit :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbReasonToVisit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbAdditionalData">
       <property name="title">
        <string> Additional data </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lblPhone">
            <property name="text">
             <string>Phone number :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledPhone"/>
          </item>
          <item>
           <widget class="QLabel" name="lblEmail">
            <property name="text">
             <string>E-Mail address :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledEmail"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="lblComment">
            <property name="text">
             <string>Comment :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="ptComment"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkIndependent">
       <property name="text">
        <string>Independent from the local studio.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbFinishLater">
       <property name="text">
        <string>Finish later ...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAttendances">
       <property name="text">
        <string>Attendances ...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>328</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ledName</tabstop>
  <tabstop>rbGenderMale</tabstop>
  <tabstop>rbGenderFemale</tabstop>
  <tabstop>deDateBirth</tabstop>
  <tabstop>ledUniqueId</tabstop>
  <tabstop>ledCountry</tabstop>
  <tabstop>ledRegion</tabstop>
  <tabstop>ledCity</tabstop>
  <tabstop>ledZip</tabstop>
  <tabstop>ledAddress</tabstop>
  <tabstop>cmbPatientOrigin</tabstop>
  <tabstop>cmbReasonToVisit</tabstop>
  <tabstop>ledPhone</tabstop>
  <tabstop>ledEmail</tabstop>
  <tabstop>ptComment</tabstop>
  <tabstop>checkIndependent</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbFinishLater</tabstop>
  <tabstop>pbAttendances</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
